small/matras.c

```diff
@@ -12,6 +12,8 @@
 #pragma intrinsic (_BitScanReverse)
 #endif
 
+#include <trivia/util.h>
```

Comment:
There's no `trivia/util.h` in the small repository.

Reply:
Defined the `MATRAS_DIV_ROUND_UP` macro locally.